core/templates/dev/head/pages/exploration_player/feedback_popup_directive.html

```diff
@@ -27,7 +27,7 @@
                ng-disabled="!feedbackText"
                translate="I18N_PLAYER_SUBMIT_BUTTON">
     </md-button>
-    <div class="checkbox" style="font-size: 0.9em; margin: 12px 0 4px 0;" ng-show="isLoggedIn">
+    <div class="checkbox" style="font-size: 0.9em; margin: 12px 0 4px 0;line-height: 1.725" ng-show="isLoggedIn">
```

Comment:
Use of inline styling is [discouraged](https://github.com/oppia/oppia/pull/6187#discussion_r254018094), you can declare a CSS class for it.

Reply:
Changed inline styling to a separate css class. @YashJipkate